src/components/Modal.js

```diff
@@ -60,8 +61,8 @@ const Modal = React.createClass({
 
   componentDidMount () {
   /*eslint-disable */
-    if (this.props.isOpen !== null) {
-      console.warn('WARNING: The prop "isOpen" is depracated in this version of the component. Please handle Modal opening from its parent.');
+    if (_has(this.props, 'isOpen')) {
```

Comment:
@lukeschunk it should work because  if `isOpen` does not exist, `this.props.isOpen` would be `undefined` which is a falsey value.